### PR TITLE
[SMALLFIX] java doc for CheckConsistencyCommand#checkConsistency

### PR DESCRIPTION
**Pull request by whatbeg**
java doc for CheckConsistencyCommand#checkConsistency

### CODE REVIEW

File: shell/src/main/java/alluxio/shell/command/CheckConsistencyCommand.java [hunk 1]
```diff
@@ -61,6 +61,14 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
     return 0;
   }
 
+  /**
+   * check consistency and find out inconsistent files.
```
Comment (gpang) [inline shell/src/main/java/alluxio/shell/command/CheckConsistencyCommand.java]: To keep the consistent style, the first word should be capitalized. It should be `Checks`.

Comment (whatbeg) [inline shell/src/main/java/alluxio/shell/command/CheckConsistencyCommand.java]: @gpang Thanks.

**Comment by whatbeg** [comment]
@uronce-cc @gpang  made some changes, please take a look.

File: shell/src/main/java/alluxio/shell/command/CheckConsistencyCommand.java [hunk 2]
```diff
@@ -61,6 +61,16 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
     return 0;
   }
 
+  /**
+   * Checks the inconsistent files and directories which exist in Alluxio but don't exist in the
+   * under filesystem under a specified persisted path, prints the results and removes them if
```
Comment (uronce-cc) [inline shell/src/main/java/alluxio/shell/command/CheckConsistencyCommand.java]: s/under filesystem/under storage

Comment (uronce-cc) [inline shell/src/main/java/alluxio/shell/command/CheckConsistencyCommand.java]: remove "under a specified persisted path"

Comment (uronce-cc) [inline shell/src/main/java/alluxio/shell/command/CheckConsistencyCommand.java]: s/prints the ..../repairs the inconsistent paths by deleting them if repairConsistency is true

**Comment by whatbeg** [comment]
@uronce-cc thanks for your comments, and I updated the javadoc. : )

**Comment by uronce-cc** [comment]
since it's a smallfix, I'm merging it now.

**Pull request merged**